hack/install.sh

```diff
@@ -49,11 +49,18 @@ do_install() {
 			;;
 	esac
 
-	if command_exists docker || command_exists lxc-docker; then
+	if command_exists docker; then
 		cat >&2 <<-'EOF'
-		Warning: "docker" or "lxc-docker" command appears to already exist.
-		Please ensure that you do not already have docker installed.
-		You may press Ctrl+C now to abort this process and rectify this situation.
+			Warning: the "docker" command appears to already exist on this system.
+
+			If you already have Docker installed, this script can cause trouble, which is
+			why we're dispalying this warning and providing the opportunity to cancel the
```

Comment:
`dispalying` hm, had to look that one up;

`dispaly`

/dɪˈspɪle/

*verb*
misspelled variation of "display". Usually a result of fast typing.